charts/buffer/templates/configmaps/proxy-apache-httpd-configmap.yaml

```diff
@@ -2,9 +2,9 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: terra-buffer-proxy-configmap
+  name: {{ .Values.name }}-proxy-configmap
   labels:
-    {{ template "buffer.labels" . }}
+    {{- include "buffer.labels" . | nindent 4 }}
```

Comment:
`nindent` is this a typo?

Reply:
nope, it's `indent` but adds a newline before, so enables neater chart syntax